policies/stable-release-updates.md

```diff
@@ -0,0 +1,91 @@
+Stable Release Updates Policy
```

Comment:
Is there somewhere in the policies where release dates are discussed.. e.g. How do I know when 3.0.2 would be released?

Reply:
It should be in an OMC roadmap. We don't have one at the moment.